index.html

```diff
@@ -204,40 +171,37 @@ <h2>Photo Gallery</h2>
 <section class="survey">
     <h2>Survey</h2>
     <form method="POST" action="https://formspree.io/mgevdyal">
-        <label>What is your name?<br>
-         <input name="name_feild" type="text" required>
-        </label>
-        <label>What is your email?<br>
-         <input name="email_feild" type="email" required>
-        </label>
-        <label>Which integer is best?<br>
-         <input name="best_integer" type="number">
-        </label>
-        <label>Which day is best?<br>
-         <input name="best_day" type="date">
-        </label>
-        <label>Which bear is best?<br>
-            <select>
-                <option disabled selected value></option>
+        <label for="name">What is your name?</label>
+        <input id="name" name="name_field" type="text" required>
+        
+        <label for="email">What is your email?</label>
+        <input id="email" name="email_field" type="email" required>
+        
+        <label for="int">Which integer is best?</label>
+        <input id="int" name="best_integer" type="number">
+
+        <label for="day">Which day is best?</label>
+        <input id="day" name="best_day" type="date">
+
+        <label for="bear">Which bear is best?</label>
+            <select name="best_bear" id="bear">
+                <option selected value></option>
                 <option value="Black">Black</option>
                 <option value="Brown">Brown</option>
                 <option value="Care">Care</option>
                 <option value="Panda">Panda</option>
                 <option value="Polar">Polar</option>
                 <option value="Teddy">Teddy</option>
             </select>
-        </label>
-        <label for="radio">Do you like radio buttons?<br>
-        <label for="yes">Yes</label>
-        <input type="radio" value="Yes" name="radio"><br>
-        <label for="No">No</label>
-        <input type="radio" value="No" name="radio"><br>
-        <label for="None of your business?">None of your business?</label>
-        <input type="radio" value="None of your business?" name="radio" checked><br>
-        </label>
-        <label>What is the meaning of life?</br>
-           <textarea name="life_meaning" rows="10" cols="100" placeholder="Enter the correct answer here"></textarea>
-        </label>
+
+        <label>Do you like radio buttons?</label>
+        <label><input type="radio" value="yes" name="radio"> Yes</label>
+        <label><input type="radio" value="no" name="radio"> No </label>
+        <label><input type="radio" value="?" name="radio" checked> None of your business</label>
```

Comment:
Corrected labels so that radio buttons are nested inside with text after the checkbox